ttnn/cpp/ttnn/operations/eltwise/unary/common/types.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
```

Comment:
We have too many types.hpp files. Maybe lets give it another name?

Reply:
Fixed